src/App.tsx

```diff
@@ -92,6 +93,9 @@ const App: React.FC = () => (
         </Switch>
       </Layout>
     </Router>
+    {process.env.NODE_ENV === 'development' &&
+      Web3Connect.isMobile() &&
```

Comment:
I tried on mobile but wasn't able to see the console.
What do I need to turn it on?

Reply:
did you try it in development or opened the pr418-- link ? the link is deployed in production

Reply:
The link from this PR (https://pr418--dexreact.review.gnosisdev.com)

Reply:
@alfetopito He's saying those PR links are always `NODE_ENV='production'` so it won't work from there

Reply:
Ahhhhh got it. So I need to spin this up in local and then open the browser? Giving it a try...

Reply:
Saw it now, very cool